### PR TITLE
chore: update json tag APY to apy

### PR DESCRIPTION
**Pull request by RafilxTenfen**
## Description
- To be compatible with the other structures and avoid casting to lowercase on rust side as well

closes: #XXXX

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by toteki** [comment]
We might also be able to change the `APY` field in the proto struct itself to lowercase - that would also make it consistent with other query response structs. (Side effect would be possibly appearing as `Apy` on the go side).

On the other hand, this json tag has the lowest impact on anything else so might be better.

Anyone else have thoughts on the best approach?

File: proto/umee/leverage/v1/query.proto [hunk 1]
```diff
@@ -181,7 +181,8 @@ message QueryBorrowAPY {
 message QueryBorrowAPYResponse {
   string APY = 1 [
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
-    (gogoproto.nullable)   = false
+    (gogoproto.nullable)   = false,
+    (gogoproto.jsontag)    = "apy"
```
[the remark this answers is not in the document]
Comment (RafilxTenfen) [reply proto/umee/leverage/v1/query.proto:185]: The official standard for JSON is actually `PascalCase` but we have everything as `snack_case` '-'

**Comment by RafilxTenfen** [comment]
> We might also be able to change the `APY` field in the proto struct itself to lowercase - that would also make it consistent with other query response structs. (Side effect would be possibly appearing as `Apy` on the go side).
> 
> On the other hand, this json tag has the lowest impact on anything else so might be better.
> 
> Anyone else have thoughts on the best approach?

Following [this approach](https://talks.golang.org/2014/names.slide#5) that @robert-zaremba shared probably it is to be called APY on the go side, but in my opinion, being the first property name as acronyms I vote for keeping `Apy`

**Comment by toteki** [comment]
I'm leaning toward what you currently have in the PR - only changing the json tag at the moment.

**Review by toteki — APPROVED** [review]

**Pull request merged**